package.json

```diff
@@ -56,7 +56,8 @@
   "scripts": {
     "build": "gulp",
     "build:test": "gulp unify",
-    "test": "karma start .config/karma.conf.js --single-run",
+    "build:dev": "gulp --dont-break",
```

Comment:
I feel like build:test and build: dev are the same. We should merge them

Reply:
gulp --dont-break is mainly for the linter. But I am all for killing `build:test`. I don't think its necessary anymore